### ISSUE TITLE
[brenchla] MkDocs exercise

### ISSUE THREAD

**Issue by LarissaBrencher** [Simulation-Software-Engineering/documentation-tools-exercise#12]
Here are the links to my [GitHub repository](https://github.com/LarissaBrencher/sse-mkdocs-exercise) and the corresponding [website](https://larissabrencher.github.io/sse-mkdocs-exercise/)

**Comment by LarissaBrencher** [comment]
@ajaust I tried to add a label, but could not select one (it was kind of greyed-out as if I am missing some roles or access rights?)

**Comment by ajaust** [comment]
You are right. It seems that only the owners of the repository can add labels. I will adapt the exercise sheet accordingly. Thanks for letting me know about the issue.

**Comment by ajaust** [comment]
Looks good.
